faraday.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.4'
 
+  spec.add_dependency 'faraday-net_http', '~> 0.0.6'
```

Comment:
Hm, perhaps we should start it at 1.0?

Reply:
Like, the code is what we use in this repo.

Reply:
Yes I have already bumped `faraday-net_http` version to 1.0, BUT I realised @JanDintel is the only owner of the gem in [Rubygems](https://rubygems.org/gems/faraday-net_http) at the moment! So I can't publish it right now.
@JanDintel could you please add me and @olleolleolle as owners as well so we can publish new versions?

Reply:
@iMacTia and @olleolleolle I've added you as owners to the gem! You should have full permission to publish a new version.

@iMacTia I noticed that you still need to set-up MFA on Rubygems though :)

Reply:
@JanDintel thank you! Will release it now then, thanks a lot for all the work 🎉 

> @iMacTia I noticed that you still need to set-up MFA on Rubygems though :)

Yes, that's a looooooong story: https://github.com/rubygems/rubygems.org/issues/2500